site/docs/v1/tech/tutorials/integrate-laravel-api.md

```diff
@@ -0,0 +1,245 @@
+```
+
+Then, you can run the setup script.
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the [PHP client library](/docs/v1/tech/client-libraries/php) documentation for more information." %}
+
+This will create the FusionAuth configuration for your {{page.technology}} API.
+
+```shell
+php setup.php "YOUR_API_KEY_FROM_ABOVE"
+```
+
+If you want, you can [log into your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} API
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
+
+First, make a directory:
+
+```shell
+mkdir ../setup-laravel && cd ../setup-laravel
+```
+
+Next, create a simple {{page.technology}} template using Laravel's build script.
+
+```shell
+curl -s "https://laravel.build/fusionauth-example-laravel-api?with=mariadb" | bash
+```
+
+This can take several minutes to complete, so please be patient.
+
+### Adding JWT Authentication
+
+After it is done, change into the `fusionauth-example-laravel-api` directory and install [`tymon/jwt-auth`](https://github.com/tymondesigns/jwt-auth), a library that adds JWT capabilities to Laravel.
+
+```shell
+cd fusionauth-example-laravel-api
+./vendor/bin/sail composer require tymon/jwt-auth
+```
+
+Add some authentication routes to `routes/api.php`. 
+
+```diff
+{% remote_include https://github.com/FusionAuth/fusionauth-example-laravel-api/commit/49a4e7c3e3610403c6a00246bf5b997bc44b5bd4.diff %}
+```
+
+Create `app/Http/Controllers/AuthController.php` to hold the authentication process.
+
+```php
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-laravel-api/main/laravel/app/Http/Controllers/AuthController.php %}
+```
+
+Laravel uses something called [Guards](https://laravel.com/docs/10.x/authentication#adding-custom-guards) to protect your endpoints, so we need to tell it about the new Guard provided from that library.
+
+```diff
+{% remote_include https://github.com/FusionAuth/fusionauth-example-laravel-api/commit/5fb7134a70e1adee39f2d1d64ce4ed5ac7303a47.diff %}
+```
+
+To make the library available for use, publish its configuration by running the command below.
+
+```shell
+./vendor/bin/sail artisan vendor:publish --provider="Tymon\JWTAuth\Providers\LaravelServiceProvider"
+```
+
+### Editing Files
+
+The Laravel installer already brings some useful resources for many applications. We'll have to edit a few of them and add some more.
+
+First, remove the need for users to have a password by editing `app/Models/User.php`.
```

Comment:
Why do we need to do this? If users have passwords, does that impact the API authentication somehow?

Reply:
I created this imagining that FusionAuth would be the single source of truth, so the only way one could access the API would be via JWT. I could make it just an additional auth provider though and accept user+password as well, then I'd just modify the `password` column to accept `NULL` values.

Reply:
Okay, let's say that. "While laravel usually expects to be able to have a username and password for authentication, APIs in general should expect an API key or a token to authenticate a request. In this quickstart, we've disabled the typical authentication method and only allow JWTs as authentication methods for this API."